loony_dev/web/static/app.css

```
@@ -0,0 +1,400 @@
+  color: var(--primary);
+}
+td.cmdline { font-family: var(--font-mono); font-size: var(--fs-1); word-break: break-all; }
+
+/* Responsive: tables collapse to cards on narrow screens. */
+@media (max-width: 560px) {
+  .data-table thead {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)